src/utils/numbers.ts

```diff
@@ -223,9 +233,9 @@ export const formatDisplayNumber = (
   // It might return number with longer fraction digits than maximumFractionDigits
```

Comment:
can we use <code>[roundingPriority](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#roundingpriority): ['lessPrecision'](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#lessprecision)</code> option for this instead of parsing and rebuilding the formatted number?

Reply:
nice point